srcpkgs/python3-pillow-simd/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'pillow-simd'
+pkgname=python3-pillow-simd
+version=5.3.0.post1
+revision=1
+archs="x86_64*"
+build_style=python3-module
+pycompile_module="PIL"
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel libjpeg-turbo-devel libopenjpeg2-devel
+ zlib-devel tiff-devel freetype-devel lcms2-devel libwebp-devel
+ libimagequant-devel freetype-devel harfbuzz-devel fribidi-devel libraqm-devel"
+short_desc="Python3 Imaging Library (PIL) fork (SIMD)"
+maintainer="mustaqim <mustaqim@pm.me>"
+license="custom: PIL"
+homepage="https://python-pillow.org/pillow-perf/"
+changelog="https://github.com/uploadcare/pillow-simd/compare/v5.2.0.post1...v5.3.0.post1"
+distfiles="https://github.com/uploadcare/${pkgname}/archive/v${version}.tar.gz"
```

Comment:
This is broken since the url does not match `pkgame` any more.

Reply:
okay, I've resolved the last of the issues